opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -714,7 +742,11 @@ namespace cpgrid
             const int* fp = output.face_ptr;
             for (int face = 0; face < num_faces; ++face) {
                 int output_face = face_to_output_face[face];
-                f2p.appendRow(fn + fp[output_face], fn + fp[output_face+1]);
+                if (output_face == cpgrid::NNCFace) {
+                    f2p.appendRow(fn, fn);
```

Comment:
This seems highly dangerous! I do not get what you are doing here. Would you mind to explain? Definitely missing at least a comment.

Reply:
I am adding an empty row, thereby associating no points with the face. I'll add a comment.